Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -55,140 +60,135 @@ class TransactionHistoryViewController: UIViewController {
     
     override func viewDidAppear(_ animated: Bool) {
         if !AppDelegate.walletLoader.isSynced {
-            print(" wallet not synced on history")
+            print("wallet not synced on history")
             return
         }
-        self.visible = true
-        if (self.FromMenu){
-            refreshControl.showLoader(in: self.tableView)
-            loadTransactions()
-            FromMenu = true
+        self.refreshControl.showLoader(in: self.tableView)
+        loadAllTransactions()
+    }
+    
+    func initFilterBtn() {
```

Comment:
This is only called once, from `viewDidLoad`. Can we move the content of this function to `viewDidLoad` instead? It's just one statement. Can add a comment there as you deem appropriate.

Reply:
fixed in 28f46d1